priv/templates/phoenix.gen.model/migration.exs

```diff
@@ -8,9 +8,8 @@ defmodule <%= base %>.Repo.Migrations.Create<%= scoped %> do
 <% end %><%= for {_, i, _, s} <- assocs do %>      add <%= if(String.ends_with?(inspect(i), "_id"), do: inspect(i), else: inspect(i) <> "_id") %>, references(<%= inspect(s) %>, on_delete: :nothing<%= if binary_id do %>, type: :binary_id<% end %>)
 <% end %>
       timestamps()
-    end
-<%= for index <- indexes do %>
-    <%= index %><% end %>
+    end<%= if length(indexes) > 0 do %>
```

Comment:
@chrismccord sorry, missed the notification :) If we move `for` up to the previous line, then the result is:

```
  def change do
    create table(:somethings) do
      add :name, :string

      timestamps()
    end
    create unique_index(:somethings, [:name])
  end
```

(Blank line between `create table(...)` section and indexes is missing.`)